clock_exercise/ClockPage.qml

```diff
@@ -5,49 +5,44 @@ Rectangle {
 
     signal timerButtonClicked
     signal alarmButtonClicked
+    signal clickOnTimerIcon
 
     color: "#151B2E"
 
-    Text {
-        x: 162
-        y: 40
-        width: 156
-        height: 36
+
+    TitleOfThePage {
+        y: 40; anchors.horizontalCenter: parent.horizontalCenter
         text: qsTr("DeveClock")
-        color: "#9FAAB0"
-        font.pixelSize: 30
-        horizontalAlignment: Text.AlignHCenter
-        font.family: "Buenos Aires"
-        font.styleName: "SemiBold"
     }
 
-    Image {
-        x: 359
-        y: 43
-        width: 30
-        height: 33
-        source: "images/timer_icon.png"
+    MouseArea {
+        x: 349; y:33
+        width: 50; height: 50
+        onClicked: clickOnTimerIcon()
+        visible: clockManager.timerCurrentSeconds > 0.0
+        enabled: clockManager.timerCurrentSeconds > 0.0
```

Comment:
Come prima, in realtà qui l'enabled non serve se stai già impostando il `visible` a false